src/phoenix/server/api/types/Model.py

```diff
@@ -37,6 +40,8 @@ def dimensions(
             [
                 to_gql_dimension(index, dimension)
                 for index, dimension in enumerate(model.scalar_dimensions)
+                if (not isinstance(include, DimensionFilter) or include(dimension))
+                and (not isinstance(exclude, DimensionFilter) or not exclude(dimension))
```

Comment:
Using `__call__` in this way feels a tad non-intuitive. defining a `matches` method feels a bit more semantic. e.g. `include.matches(dimension)`

Reply:
I think it boils down to the fact that the variable names for these filters are almost always verbs. Below would be a situation where `__call___` would be inadequate, but as long as the variable names are verbs, I think we can opt for '__call__' for the benefit of concision.
```python
zzz(dimension)
```
vs
```python
zzz.matches(dimension)
```

Reply:
concision is definitely over-rated when you have multiple people contributing to a codebase. I rely heavily on code completion to describe what is possible for a given class or object. If you rely on the naming of a variable to make it readable, you are getting a too fancy. If you alias it, maybe that's okay but  I believe we should aim to make un-intuitive code impossible to write. A library needs to be progressively learnable and if there's too much magic, you make it hard to onboard.

Reply:
ok my argument was bad, but ultimately this is not anything fancy, because in python functions are first class objects, and the usage here is a reflection of the functional programming paradigm